.pre-commit-config.yaml

```diff
@@ -71,12 +71,12 @@ repos:
         entry: python3 ./cobalt/precommit/python3_check.py
         language: python
         types: [python]
-    -   id: google-java-format
-        name: google-java-format
-        entry: python3 ./cobalt/precommit/google_java_format_wrapper.py
-        language: python
-        types: [java]
-        args: [-i]
+    # -   id: google-java-format
```

Comment:
I wouldn't comment this out, instead use the `exclude:` field for cobalt and starboard directories (see an example below in gcheckstyle)